js/datetime/Index.svelte

```diff
@@ -10,6 +10,7 @@
 	export let gradio: Gradio<{
 		change: undefined;
 		submit: undefined;
+		clear: undefined;
```

Comment:
where do you dispatch this?

Reply:
I don't think it makes sense to add a clear event listener as such. We don't have it in any other "form" type components (textbox, number, dropdown) , they only exist in media type components (image, audio, video) and are meant to signify when an uploaded media type is reset by clicking the X.

Reply:
Agreed, I think the original issue was referencing clicking the "clear" button in `gr.Interface`. Since there's no dedicated clear button in `gr.Datetime`, we don't need an event for it